etape1/src/main/java/org/iae/annecy/st1/etape1/view/MenuView.java

```diff
@@ -0,0 +1,128 @@
+package org.iae.annecy.st1.etape1.view;
+
+import org.iae.annecy.st1.tools.ConsoleHelper;
+
+public class MenuView{
+
+    /**
+     * Affiche les menus principaux suivant le rôle de la personne
+     * 
+     * @param cat
+     * 		le nombre récupéré par le scanner
+     * 		===> compris entre 1 et 4
+     * 
+     * @return Le menu principal sous forme de chaines de caractères
+     */
+    public void affichageMenu(Integer cat){
+	String message = null;
+	switch(cat){
+	    case 1:
+		message = "**************MENU PRINCIPAL**************\n"
+			+ "\t1. Modifier un produit\n"
+			+ "\t2. Ajouter un produit\n"
+			+ "\t3. Supprimer un produit\n"
+			+ "\t4. Afficher le catalogue\n";
+		break;
+	    case 2:
+		message = "**************MENU PRINCIPAL**************\n"
+			+ "\t1. Modifier un client\n"
+			+ "\t2. Ajouter un client\n"
+			+ "\t3. Supprimer un client\n"
+			+ "\t4. Afficher la liste des clients\n";
+		break;
+	    case 3: 
+		message = "**************MENU PRINCIPAL**************\n"
+			+ "\t1. Ajouter un produit au panier\n"
+			+ "\t2. Supprimer un produit du panier\n"
+			+ "\t3. Voir le panier\n"
+			+ "\t4. Valider la commande";
+		break;
+	    case 4:
+		message = "**************MENU PRINCIPAL**************\n"
+			+ "\t1. Valider une commande\n"
+			+ "\t2. Consulter la liste des produits achetés";
+		break;
+	    default:
+		this.messageErreur();

```

Comment:
pour rester "consistant" avec les `case` du `switch`, mais dans la variable "message" le texte renvoyé par this.messageErreur() (en modifiant cette méthode pour quel renvoie une String, plutôt que l'afficher directement).